waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/client/TunnelingMetaStoreClientFactory.java

```diff
@@ -112,12 +80,13 @@ public CloseableThriftHiveMetastoreIface newInstance(HiveConf hiveConf, String n
     TunnelableFactory<CloseableThriftHiveMetastoreIface> tunnelableFactory = tunnelableFactorySupplier
         .get(localHiveConf);
 
-    LOG.info("Metastore URI {} is being proxied through {}", hiveConf.getVar(HiveConf.ConfVars.METASTOREURIS),
-        localHiveConf.getVar(HiveConf.ConfVars.METASTOREURIS));
+    LOG
+        .info("Metastore URI {} is being proxied through {}", hiveConf.getVar(HiveConf.ConfVars.METASTOREURIS),
```

Comment:
Formatting

Reply:
I can't change this :(
Eclipse formatter just splits it like that and if I remove it manually, the next time I save the file, Eclipse puts the line break back there.

Reply:
OK, a bit weird, but let's not worry about this now, we're probably going to need to review some of our formatting rules soon and can check what's going with things like this then.